### PR TITLE
Add multiple links

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
Addresses https://github.com/NASA-IMPACT/delta-ui/issues/293

This is not fancy in any way but curious if this approach is what others were thinking to include multiple ways to open the notebooks

Screen Shot 2022-11-07 at 3 57 26 PM

### CODE REVIEW

**Review by nerik — APPROVED** [review]
Looks good to me 🚢 
Just did a little house cleaning.

I guess we should also update the content repo ([here](https://github.com/NASA-IMPACT/delta-config/blob/develop/datasets/no2.data.mdx?plain=1#L7) and [here](https://github.com/NASA-IMPACT/delta-config/blob/develop/datasets/hls-events.ej.data.mdx?plain=1#L5))

File: app/scripts/components/datasets/s-usage/index.js [hunk 1]
```diff
@@ -45,11 +45,18 @@ function DatasetsUsage() {
...
+                Check out how to use this dataset:{' '}
+                <ul>
+                  {datasetUsages.map((datasetUsage) => (
+                    <li key={datasetUsage.url}>
+                      {datasetUsage.description}:{' '}
+                      <a href={datasetUsage.url}>{datasetUsage.title}</a>
+                    </li>
+                  ))}
+                </ul>
               </p>
```
Comment (danielfdsilva) [inline app/scripts/components/datasets/s-usage/index.js:60]: A list can't be a children of `p`. Move it after the closing tag

File: mock/datasets/no2.data.mdx [hunk 2]
```diff
@@ -5,8 +5,12 @@ featuredOn:
   - covid-19
 description: "Since the outbreak of the novel coronavirus, atmospheric concentrations of nitrogen dioxide have changed by as much as 60% in some regions."
 usage:
-  url: 'https://nasa-impact.github.io/veda-documentation/timeseries-stac-api.html'
-  title: 'Time series using STAC API statistics endpoints'
+  - url: 'https://nasa-impact.github.io/veda-documentation/timeseries-stac-api.html'
+    description: Static notebook
```
Comment (danielfdsilva) [inline mock/datasets/no2.data.mdx:9]: No 100% sold on the `description` property. Normally it denotes something longer and it is used as such in other parts of the app. I'd say this is more of a `label` perhaps?

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

File: app/scripts/components/datasets/s-usage/index.js (hunk 1 above)
Comment (abarciauskas-bgse) [reply app/scripts/components/datasets/s-usage/index.js]: Done in https://github.com/NASA-IMPACT/delta-ui/pull/311/commits/3db9861271a2b0eac3a1a0db1acf827916d6c8d3

File: mock/datasets/no2.data.mdx (hunk 2 above)
Comment (abarciauskas-bgse) [reply mock/datasets/no2.data.mdx]: Done!

**Review by danielfdsilva — APPROVED** [review]
@abarciauskas-bgse Thank you for the fixes.

Did some housekeeping:
- Fix linting errors
- Rebase to remove the self merge conflict.

If anything else is needed on this branch be sure to do a `git reset`

**Review by danielfdsilva — APPROVED** [review]
Good to go. Only thing left is the documentation as @nerik mentioned

**Comment by abarciauskas-bgse** [comment]
Thanks @danielfdsilva @nerik 

I added the documentation task as a checklist item in #293

**Pull request merged**